recipes/libiconv/all/conanfile.py

```diff
@@ -55,99 +63,72 @@ def configure(self):
     def _settings_build(self):
         return getattr(self, "settings_build", self.settings)
 
-    def build_requirements(self):
-        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires("msys2/cci.latest")
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+    def generate(self):
```

Comment:
would be nice to move this after source(), otherwise recipe execution flow is a little bit confusing